core/src/operations/mod.rs

```diff
@@ -426,100 +425,100 @@ impl Operation {
+            Self::MovUp6        => 0b0001_0100,
+            Self::MovDn6        => 0b0001_0101,
+            Self::MovUp7        => 0b0001_0110,
+            Self::MovDn7        => 0b0001_0111,
+            Self::SwapW         => 0b0001_1000,
+            Self::Ext2Mul       => 0b0001_1001,
+            Self::MovUp8        => 0b0001_1010,
+            Self::MovDn8        => 0b0001_1011,
+            Self::SwapW2        => 0b0001_1100,
+            Self::SwapW3        => 0b0001_1101,
+            Self::SwapDW        => 0b0001_1110,
+            // <empty>          => 0b0001_1111,
+
+            Self::Assert        => 0b0010_0000,
+            Self::Eq            => 0b0010_0001,
+            Self::Add           => 0b0010_0010,
+            Self::Mul           => 0b0010_0011,
+            Self::And           => 0b0010_0100,
+            Self::Or            => 0b0010_0101,
+            Self::U32and        => 0b0010_0110,
+            Self::U32xor        => 0b0010_0111,
+            Self::FriE2F4       => 0b0010_1000,
+            Self::Drop          => 0b0010_1001,
+            Self::CSwap         => 0b0010_1010,
+            Self::CSwapW        => 0b0010_1011,
+            Self::MLoadW        => 0b0010_1100,
+            Self::MStore        => 0b0010_1101,
+            Self::MStoreW       => 0b0010_1110,
+            Self::FmpUpdate     => 0b0010_1111,
+
+            Self::Pad           => 0b0011_0000,
+            Self::Dup0          => 0b0011_0001,
+            Self::Dup1          => 0b0011_0010,
+            Self::Dup2          => 0b0011_0011,
+            Self::Dup3          => 0b0011_0100,
+            Self::Dup4          => 0b0011_0101,
+            Self::Dup5          => 0b0011_0110,
+            Self::Dup6          => 0b0011_0111,
+            Self::Dup7          => 0b0011_1000,
+            Self::Dup9          => 0b0011_1001,
+            Self::Dup11         => 0b0011_1010,
+            Self::Dup13         => 0b0011_1011,
+            Self::Dup15         => 0b0011_1100,
+            Self::Read          => 0b0011_1101,
+            Self::SDepth        => 0b0011_1110,
+            Self::Clk           => 0b0011_1111,
+
+            Self::U32add        => 0b0100_0000,
+            Self::U32sub        => 0b0100_0010,
+            Self::U32mul        => 0b0100_0100,
+            Self::U32div        => 0b0100_0110,
+            Self::U32split      => 0b0100_1000,
+            Self::U32assert2    => 0b0100_1010,
+            Self::U32add3       => 0b0100_1100,
+            Self::U32madd       => 0b0100_1110,
+
+            Self::HPerm         => 0b0101_0000,
+            Self::MpVerify      => 0b0101_0010,
+            Self::Pipe          => 0b0101_0100,
+            Self::MStream       => 0b0101_0110,
+            Self::Span          => 0b0101_1000,
+            Self::Join          => 0b0101_1010,
+            Self::Split         => 0b0101_1100,
+            Self::Loop          => 0b0101_1110,
+
+            Self::MrUpdate(_)   => 0b0110_0000,
```

Comment:
I think we should be able to have this just as a simple enum variant (with no internal values).

Reply:
Removed in favor of advice injector